modules/local/openms_featurefinderidentification.nf

```diff
@@ -40,11 +27,10 @@ process OPENMS_FEATUREFINDERIDENTIFICATION  {
         FeatureFinderIdentification -in $mzml \\
             -out ${prefix}.featureXML \\
             -threads $task.cpus \\
-            ${arguments} \\
-            $options.args
```

Comment:
Its totally fine in case you do not need the additional args by `$options.args`, but if you do, you could add them by using `arguments = task.ext.args ? "${task.ext.args} -id $id_quant" : "-id $id_quant"` instead of `arguments = "-id $id_quant"` or such, a few lines above.